{
  "model": "gpt-4-1",
  "service": "github",
  "task": "harmony__fix_conflict",
  "runs": {
    "run-1": {
      "success": false,
      "error_message": null,
      "execution_time": 93.71423816680908,
      "token_usage": {
        "input_tokens": 564374,
        "output_tokens": 1136,
        "total_tokens": 565510,
        "reasoning_tokens": 0
      },
      "turn_count": 14
    },
    "run-2": {
      "success": false,
      "error_message": null,
      "execution_time": 91.05192303657532,
      "token_usage": {
        "input_tokens": 1304329,
        "output_tokens": 1575,
        "total_tokens": 1305904,
        "reasoning_tokens": 0
      },
      "turn_count": 21
    },
    "run-3": {
      "success": false,
      "error_message": null,
      "execution_time": 64.31709241867065,
      "token_usage": {
        "input_tokens": 899887,
        "output_tokens": 1251,
        "total_tokens": 901138,
        "reasoning_tokens": 0
      },
      "turn_count": 15
    },
    "run-4": {
      "success": false,
      "error_message": null,
      "execution_time": 71.99849033355713,
      "token_usage": {
        "input_tokens": 786169,
        "output_tokens": 1267,
        "total_tokens": 787436,
        "reasoning_tokens": 0
      },
      "turn_count": 19
    }
  }
}